<launch>
    <node name="object_visualizer" pkg="object_visualizer" type="object_visualizer_node" output="screen">
        <param name="object_file_path" value="$(find object_visualizer)/data/object_detection.csv"/>
        <param name="map_frame" value="map"/>
        <param name="text_scale" value="0.5"/>
        <param name="marker_scale" value="0.5"/>
    </node>

    <!-- <node name="bounding_box_publisher" pkg="object_visualizer" type="cluster_visualizer_node" output="screen">
        <param name="cluster_file_path" value="$(find object_visualizer)/data/cluster_centers.csv"/>
        <param name="map_frame" value="map"/>
    </node> -->

    <node name="object_clustering_node" pkg="object_visualizer" type="object_clustering_node" output="screen">
        <param name="input_csv" value="$(find object_visualizer)/data/object_detection.csv"/>
        <param name="output_csv" value="$(find object_visualizer)/data/clustered.csv"/>
        <param name="cluster_distance_threshold" value="1.0"/>
        <param name="map_frame" value="map"/>
        <param name="text_scale" value="0.5"/>
        <param name="marker_scale" value="0.5"/>
    </node>
    <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find object_visualizer)/rviz/view_objects.rviz" output="screen"/> -->
</launch>